packages/algob/test/builtin-tasks/run.ts

```diff
@@ -170,7 +196,9 @@ scripts directory: script 1 executed
   it("Should crash on deployASC", async function () {
     await expectBuilderErrorAsync(
       async () =>
-        await this.env.run(TASK_RUN, { scripts: ["scripts/other-scripts/deploy-asc.js"] }),
+        await this.env.run(TASK_RUN, {
+          scripts: ["scripts/other-scripts/deploy-asc.js"]
+        }),
       ERRORS.BUILTIN_TASKS.DEPLOYER_EDIT_OUTSIDE_DEPLOY,
       "deployASC"
     );
```

Comment:
Are you planning to add a test which will use a direct call for a task function, rather than `cli`?

Reply:
This doesn't call the chain because run is not allowed to do it because of `isWriteable` switch.